lua/lazyvim/plugins/extras/lang/vue.lua

```diff
@@ -19,7 +19,14 @@ return {
     "neovim/nvim-lspconfig",
     opts = {
       servers = {
-        volar = {},
+        volar = {
+          filetypes = { "vue" },
```

Comment:
That is already default for volar LSP server (see [here](https://github.com/neovim/nvim-lspconfig/blob/37f362ef42d1a604d332e8d3d7d47593852b4313/lua/lspconfig/server_configurations/volar.lua#L18)). So, maybe not needed?

Reply:
Because vtsls has dealt with ts and js, I think it may be better for volar to only deal with vue.

Since the default configuration only includes vue, it is indeed duplicated here, and it's no problem to delete it.